common/tools/turborepo-remote-cache/src/auth.ts

```diff
@@ -7,6 +7,9 @@ import process from "node:process";
 // Token authentication as a Fastify plugin
 export const authenticateToken: FastifyPluginAsync = async (fastify) => {
   fastify.addHook("preHandler", async (request: FastifyRequest, reply: FastifyReply) => {
+    if (process.env.AZURE_CONTAINER_SAS_URL) {
```

Comment:
Why would we not want to check this if we have a SAS container URL?  We would still have a bearer token from Turbo itself.

Reply:
I was thinking if one has permissions to the container via SAS url, turbo token is less useful now, as it was used to authenticate with the cache server then cache server provides access to containers.  But if turbo token still has other purpose, then we can keep checking for it.

Reply:
No, it's more like if you have permission set on the team to be able to write which comes from your config and matches one available in the server setup.

Reply:
reverted